src/azure-cli/azure/cli/command_modules/appconfig/feature.py

```diff
@@ -293,10 +295,21 @@ def list_feature(cmd,
                                                    key_filter=key_filter,
                                                    label=label if label else SearchFilterOptions.ANY_LABEL)
         retrieved_featureflags = []
+
+        invalid_ffs = 0
         for kv in retrieved_keyvalues:
-            retrieved_featureflags.append(
-                map_keyvalue_to_featureflag(
-                    keyvalue=kv, show_conditions=True))
+            try:
+                retrieved_featureflags.append(
+                    map_keyvalue_to_featureflag(
+                        keyvalue=kv, show_conditions=True))
+            except (CLIErrors.InvalidArgumentValueError, ValueError, TypeError) as exception:
+                logger.warning("%s\n", exception)
```

Comment:
The error is already being logged in map_keyvalue_to_featureflag. Do we need to log here again?

Reply:
Only the unknown exceptions which are not caught here are logged in `map_keyvalue_to_featureflagvalue`. Could I be missing something?